lib/props/index.js

```diff
@@ -0,0 +1,9 @@
+import _curry2 from '../_internal/_curry2'
+import prop from '../prop'
+
+export default _curry2(function props(path, object) {
+  if (!Array.isArray(path)) throw new TypeError('Path should be an array')
+  return path.map(function(key) {
```

Comment:
It could be written through `for` loop, and I think it could be faster and also lighter if not to use `import prop`

Reply:
need to check